experiment/simulation/js/main.js

```diff
@@ -1 +1,586 @@
+    ];
+  } else if (choice === "ir") {
+    document.getElementById("device-spectro").innerHTML = "Desiccator";
+
+    document.getElementById("spectro").src = "./images/Desiccator.png";
+    document.getElementById("spectro").style.minHeight = "600px";
+    document.getElementById("caption").innerHTML = "Desiccator";
+    document.getElementById("vid-source").src = "./images/IRAnimation.mp4";
+    document.getElementById("animation-bottom-right").load();
+
+    setupMessages[3] =
+      "Click on the Desiccator option in the Apparatus Menu to introduce it into the workspace.";
+
+    instructionMessages = [
+      "Click on the Sample beaker to transfer some amount of the sample into the Tube.",
+      "Click on the Tube to place it into the Desiccator.",
+    ];
+
+    observationMessages = [
+      "Now observe the zoomed in animation of the Desiccator. The ray goes from the IR Source to the Beam Splitter, from where it further proceeds to the Fixed and Movable Mirrors. The reflected rays are merged at the Beam Splitter and proceed towards the Detector for detection.",
+      "The points on the graph are used to depict the Intensity of the ray detected.",
+    ];
+  } else if (choice === "nmr") {
+    document.getElementById("device-spectro").innerHTML = "NMR Spectrometer";
+
+    document.getElementById("spectro").src = "./images/NMRSpectro.png";
+    document.getElementById("caption").innerHTML = "NMR Spectrometer";
+    document.getElementById("vid-source").src = "./images/NMRAnimation.mp4";
+    document.getElementById("animation-bottom-right").load();
+
+    setupMessages[3] =
+      "Click on the NMR Spectrometer option in the Apparatus Menu to introduce it into the workspace.";
+
+    instructionMessages = [
+      "Click on the Sample beaker to transfer some amount of the sample into the Tube.",
+      "Click on the Tube to place it into the NMR Spectrometer.",
+    ];
+
+    observationMessages = [
+      "Now observe the zoomed in animation of the NMR Spectrometer. It can be seen that the NMR Tube is rotated between Radio Frequency Generator and Radio Frequency Receiver while the Magnetic Field passes through it.",
+      "The points on the graph are used to depict the Chemical Shifts.",
+    ];
+  }
+}
+
+async function moveTube() {
+  let image = document.getElementById("tube");
+  image.setAttribute("opacity", "1");
+  let a1 = anime.timeline({
+    targets: "#tube",
+    duration: 800,
+    easing: "linear",
+  });
+  let transX = 630;
+  let transY = -10;
+  let sc = 0.4;
+  screenWidth();
+  if (divWidth < 769) {
+    transX = -10;
+    transY = 500;
+  }
+  if (overallIteration === 2) {
+    a1.add({
+      duration: 1000,
+      translateX: transX,
+      translateY: transY,
+      scale: sc,
+    })
+      .add({
+        opacity: 0,
+      })
+      .add({
+        translateX: 0,
+        translateY: 0,
+        scale: 1,
+      });
+
+    if (choice === "nmr") {
+      document.getElementById("instruction").innerHTML =
```

Comment:
This approach for using 2 html elements to show the instructions is very confusing and extremely error prone. Please put extensive comments about why you chose this approach and how it works. Otherwise it will become very difficult for anyone to maintain this experiment.